lambdas/time-epoch-handler/tests/time-handler.test.ts

```diff
@@ -0,0 +1,200 @@
+import { TimeHandler } from "../src/time-handler";
+import { TimeUnits } from "../src/utils/time-units";
+import { Context } from "aws-lambda";
+
+const monday31st2021InMilliseconds = 1622502000000;
+const monday31st2021InSeconds = 1622502000;
+const timeHandler = new TimeHandler();
+const govJourneyId = "test-government-journey-id";
+
+jest.spyOn(Date, "now").mockReturnValue(monday31st2021InMilliseconds);
+
+describe("time-handler", () => {
+  describe("epochMode: claim - return epoch (not before) nbf and (expiry) exp for claim set", () => {
```

Comment:
[nit] not really sure what we mean by `epochMode`? Also please could we use describe blocks to scope tests rather than repeating prefixes